### PR TITLE
Invitees

### LINKED ISSUE #31: Create Invitees Component
Create the Invitee Component. This includes the photo and name, for now we can use a basic avatar until we add in code for storing profile images. 

The component will have 3 different states: 
1. No response - just the image no extra icons
2. Attending - has the blue checkmark
3. Not attending - has the red "x"

Invitees Component

### PR DESCRIPTION
**Pull request by jam1023**
Created invitees component. Takes text (name of invitee) and rsvpStatus (accepted, declined, pending).

### CODE REVIEW

**Comment by jam1023** [comment]
Screen Shot 2020-10-15 at 5 55 13 PM

File: client/src/components/Main/Invitee/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+const ImgFace = styled.img`
+width: 100%;
+position: absolute;
+right:0;
+top:0;
+`;
+
+//rsvpStatus: accepted, declined, pending
+const Invitee = ({color,text,onClick,rsvpStatus,...props}) => {
```
Comment (hokansom) [inline client/src/components/Main/Invitee/index.js]: For once, I don't think we need an onClick method, so you can get rid of that, as well as color (unless you're using it elsewhere). 

To be more specific, could you change "text" to "name."

**Review by hokansom — CHANGES_REQUESTED** [review]
You're getting really good at this! Just minor change to remove code we don't need as mentioned in the comments, and you'll be good to go!

**Comment by jam1023** [comment]
changed text to name, got rid of onClick, and removed my development lines from Main/index.js

**Review by hokansom — APPROVED** [review]
Perfect 👌

**Pull request merged**